README.md

```diff
@@ -20,36 +20,38 @@ Installation
 
 Quick start
 -----------
-- If you are interested in exporting metrics data from Sysdig Cloud, take a look at [examples/get_data_simple.py](examples/get_data_simple.py) and [examples/get_data_advanced.py](examples/get_data_advanced.py).
+- If you are interested in exporting metrics data from Sysdig Monitor, take a look at [examples/get_data_simple.py](examples/get_data_simple.py) and [examples/get_data_advanced.py](examples/get_data_advanced.py).
 - If you want to programmatically create an alert, refer to [examples/create_alert.py](examples/create_alert.py)
 - If you want to programmatically create a dashboard, refer to [examples/create_dashboard.py](examples/create_dashboard.py)
 
 Usage
 -----
 
-_Note:_ in order to use this API you must obtain a Sysdig Cloud token. You can get your user's token in the _Sysdig Cloud API_ section of the [settings](https://app.sysdigcloud.com/#/settings/user) page.
+_Note:_ in order to use this API you must obtain a Sysdig Monitor/Secure API token. You can get your user's token in the _Sysdig Monitor API_ section of the settings page for [monitor](https://app.sysdigcloud.com/#/settings/user) or [secure](https://secure.sysdig.com/#/settings/user).
 
-The library exports a single class, `SdcClient`, that is used to connect to Sysdig Cloud and execute actions. It can be instantiated like this:
+The library exports two classes, `SdMonClient` and `SdSecureClient` that are used to connect to Sysdig Monitor/Secure and execute actions. They can be instantiated like this:
 
 ``` python
-from sdcclient import SdcClient
+from sdcclient import SdMonClient
 
-sdc_token = "MY_API_TOKEN"
+api_token = "MY_API_TOKEN"
 
 #
-# Instantiate the SDC client
+# Instantiate the Sysdig Monitor client
 #
-sdclient = SdcClient(sdc_token)
+client = SdMonClient(api_token)
 ```
 
+For backwards compatibility purposes, a third class `SdcClient` is exported which is an alias of `SdMonClient`.
+
 Once instantiated, all the methods documented below can be called on the object.
 
 ####Return Values
```

Comment:
This wasn't your doing, but as long as you're in the code, might as well fix the fact that this "Return Values" bit doesn't render into a Markdown header.